### PR TITLE
Update index.js

### LINKED ISSUE #3688: [DOCS] documentation
### Description

**Getting started is small** as compare to **Popular user Guides**

### Screenshots

getting started

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Vaibhavsah1122003**
`Closes #3688 `

## Fixes Issue

Closes #3688  

## Changes proposed

Updated the pages/index.js 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Before
1st contro before

## After
1st contro after

## Note to reviewers

For some reason or other the H2 tag was not working in next.js

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3964"></a>

### CODE REVIEW

**Review by Njong392 — DISMISSED** [review]
Looks a lot more readable to me now. Thank you for the change! 🎉

File: pages/index.js [hunk 1]
```diff
@@ -395,7 +395,7 @@ export default function Home({ total, today }) {
 
         <div className="relative bg-white py-24 sm:py-32 lg:py-40">
           <div className="mx-auto max-w-md px-6 text-center sm:max-w-3xl lg:max-w-7xl lg:px-8">
-            <h2 className="text-lg font-semibold text-indigo-600">
+            <h2 className="text-lg font-semibold text-indigo-600 text-3xl">
```
Comment (eddiejaoude) [inline pages/index.js:398]: There is duplicate text size, `text-lg` and `text-3xl`

Comment (Vaibhavsah1122003) [reply pages/index.js:398]: Will take care of it

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Pull request merged**